includes/admin/class-wcmp-export.php

```diff
@@ -686,8 +685,16 @@ public static function getRecipientFromOrder(WC_Order $order)
 
                     $address_intl["street"]        = (string) $address_parts["street"];
                     $address_intl["number"]        = (string) $address_parts["number"];
-                    $address_intl["number_suffix"] = array_key_exists("number_suffix", $address_parts) // optional
-                        ? (string) $address_parts["number_suffix"] : "";
+                    $address_intl["number_suffix"] = (string) $address_parts["extension"] ?: "";
+
+
+                    if (!$address_intl["number_suffix"]) {
+                       if (preg_match("~^([a-z]{1}\d{1,3}|-\d{1,4}\d{2}\w{1,2}|[a-z]{1}[a-z\s]{0,3})(?:\W|$)~i", $address["street_additional_info"])) {
```

Comment:
Zou je van de regex geen constante kunnen maken en die hier dan aanroepen. Mocht je dezelfde regex nog eens willen gebruiken dan kan dat makkelijker. Daarbij kan je met een regex precies aangeven waarvoor hij bedoeld is.